tests/cursorid/cursorid-set_state-001.phpt

```diff
@@ -0,0 +1,21 @@
+--TEST--
+MongoDB\Driver\CursorId::__set_state()
+--FILE--
+<?php
+
+$cursorId = unserialize('C:23:"MongoDB\Driver\CursorId":42:{a:1:{s:2:"id";s:19:"7250031947823432848";}}');
+
+var_export(MongoDB\Driver\CursorId::__set_state([
+    'id' => "$cursorId",
+]));
+echo "\n\n";
+
+?>
+===DONE===
+<?php exit(0); ?>
+--EXPECTF--
+MongoDB\Driver\CursorId::__set_state(array(
+%w'id' => '7250031947823432848',
```

Comment:
You can use three spaces here (as you did in `cursorid-var_export-001.phpt`). A wildcard wouldn't really be needed unless different PHP versions started using varying indentation levels (not an issue AFAIK). Removing `%w` also means we can change `EXPECTF` to `EXPECT`.